native/src/routes/landing/components/CitySelector.js

```diff
@@ -54,6 +54,8 @@ class CitySelector extends React.PureComponent<PropsType> {
     const cities = this.props.cities
     if (normalizedFilter === 'wirschaffendas') {
       return cities.filter(_city => !_city.live)
+    } else if (__DEV__) {
```

Comment:
When is `__DEV__` set/present?

Reply:
Should be replaced with build config, closing this comment